src/ripple/test/jtx/impl/Env.cpp

```diff
@@ -537,9 +537,23 @@ Env::st (JTx const& jt)
 Json::Value
 Env::do_rpc(std::vector<std::string> const& args)
 {
-    auto const jv = cmdLineToJSONRPC(args, journal);
-    return client().invoke(jv["method"].asString(),
-        jv["params"][0U]);
+    auto jv = cmdLineToJSONRPC(args, journal);
+#if 1  // force use of 2.0 API
+    if (!jv.isMember("jsonrpc"))
+    {
+        jv["jsonrpc"] = "2.0";
+        jv["ripplerpc"] = "2.0";
+        jv["id"] = 5;
+    }
+#endif
+    auto response = client().invoke(jv["method"].asString(), jv["params"][0U]);
+    if (jv.isMember("jsonrpc"))
```

Comment:
It should and does.  However since I'm not yet sure if we're going to permanently use 2.0 in these tests, I only checked that fact with print statements instead of making the more extensive check with asserts.